### DISCUSSION TITLE
how do i get the galaxy brain badge

### DISCUSSION THREAD

**Discussion by codespako** [community/community#30697] [Other Feature Feedback, Questions, & Ideas] [answered]
how do i get the galaxy brain badge??

**Comment by n1kkec** [comment, marked as answer]
You have to have 2 accepted answers to a discussion to unlock the Galaxy Brain achievement badge.

**Comment by DenverCoder1** [comment]
By answering discussions and having your answer marked as accepted.

Note that marking your own answer as accepted does not count, it has to be by the asker or a maintainer.

It is first unlocked after 2 accepted answers and additional levels at 8, 16, and 32.

You can find more info here: https://github.com/Schweinepriester/github-profile-achievements

**Comment by github-staff** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
